src/main.rs

```diff
@@ -149,9 +144,24 @@ fn parse<T: BufRead>(
                 match serde_json::from_str(&line) {
                     Ok(event) => event,
                     Err(_) => {
-                        // Assume this line isn't part of the test result output. The test itself may
-                        // have printed e.g. a log starting with '{'
-                        continue;
+                        // Our logging inserts logs that look like json, but aren't. If we can't parse the line as json
+                        // try splitting it at "{" and taking the rightmost section.
+                        // We want the last part of the split as cargo llvm-cov returns a new line after each test json.
```

Comment:
Just `cargo test` really, nothing about this is specific to `cargo llvm-cov`. Might also be worth noting that this works because the test log itself is guaranteed to never contain a `{` (right?).